apps/vaporgui/PGroup.cpp

```diff
@@ -8,6 +8,7 @@ PGroup::PGroup()
 {
     QVBoxLayout *layout = new QVBoxLayout;
     layout->setMargin(0);
+    layout->addStretch();
```

Comment:
The PGroup widget has had layout options added. I don’t believe PWidgets should perform layout options, correct?

Reply:
Unfortunately PWidgets are QWidgets themselves, so they do need to perform some layout management.  The addStretch() function handles resizing operations on the control panel.